### PR TITLE
Use the same analysis chain to StandardAnalyzer (a follow-up of #480)

### PR DESCRIPTION
**Pull request by mocobeta**
This mimics the StandardAnalyzer.

https://github.com/apache/lucene/pull/480#pullrequestreview-817120063

### CODE REVIEW

File: lucene/luke/src/java/org/apache/lucene/luke/models/analysis/AnalysisImpl.java [hunk 1]
```diff
@@ -154,7 +158,12 @@ public Analyzer createAnalyzerFromClassName(String analyzerType) {
 
   private Analyzer defaultAnalyzer() {
     try {
-      return CustomAnalyzer.builder().withTokenizer("standard").build();
+      Map<String, String> params = new HashMap<>();
+      params.put("maxTokenLength", Integer.toString(StandardAnalyzer.DEFAULT_MAX_TOKEN_LENGTH));
+      return CustomAnalyzer.builder()
+          .withTokenizer(StandardTokenizerFactory.NAME, params)
```
Comment (uschindler) [inline lucene/luke/src/java/org/apache/lucene/luke/models/analysis/AnalysisImpl.java:164]: You don't need a map. Customanalyzer takes a Varars of strings. We should use the code in the "official way" (see examples in Javadocs).

```
return CustomAnalyzer.builder()
          .withTokenizer(StandardTokenizerFactory.NAME, params), "maxTokenLength", Integer.toString(StandardAnalyzer.DEFAULT_MAX_TOKEN_LENGTH))...
```

Comment (uschindler) [reply lucene/luke/src/java/org/apache/lucene/luke/models/analysis/AnalysisImpl.java:164]: Here's a quite complex example by Robert. Please also note the recommend formatting:

https://github.com/uschindler/german-decompounder#lucene-api-example

**Review by uschindler — CHANGES_REQUESTED** [review]
I left a minor comment on formatting and the recommend usage (no HashMaps needed).

My question (haven't tried): when you do it like that would the user interface also prefilled? If not, we can simply use StandardAnalyzer and don't need to use this code.

Otherwise I am ok.

**Comment by uschindler** [comment]
Hi,
I did some quick tests. I was expecting to see the standard analyzer defaults already predefined in the UI.
As this is not the case, I would stay with `new StandardAnalyzer()`. It is easier and useful.

I had some other idea: You could also add a line to enter the class name of an Analyzer and add a button "Instantiate class (requires no-arg constructor)".

**Comment by mocobeta** [comment]
I replaced the default to `StandardAnalyzer`.

> I did some quick tests. I was expecting to see the standard analyzer defaults already predefined in the UI.
> As this is not the case, I would stay with new StandardAnalyzer(). It is easier and useful.

Yes - while StandardAnalyzer is already used as the default (in 9.0), that is defined in the UI layer. My intention here is that set the default in the model layer; though the difference is not visible in the UI, this is more manageable and test-friendly. 

> I had some other idea: You could also add a line to enter the class name of an Analyzer and add a button "Instantiate class (requires no-arg constructor)".

This is definitely possible, but some work is needed (adding a UI component requires layout rebalancing and proper event handling); can come later.

Here is the default look of the Luke Analysis UI with the patch (no difference from 9.0, in terms of the user interface).
Screenshot from 2021-11-28 08-32-48

I'll push it to the main soon.

File: lucene/luke/src/java/org/apache/lucene/luke/models/analysis/AnalysisImpl.java (hunk 1 above)
Comment (mocobeta) [reply lucene/luke/src/java/org/apache/lucene/luke/models/analysis/AnalysisImpl.java]: > You don't need a map. Customanalyzer takes a Varars of strings. We should use the code in the "official way" (see examples in Javadocs).

I know the builder interface with variable arguments. Ok, I'll keep in mind that.

**Comment by uschindler** [comment]
Thanks 👍

**Pull request merged**